app/src/main/java/com/nmakademija/nmaakademija/adapter/ScheduleAdapter.java

```diff
@@ -1,69 +1,74 @@
 package com.nmakademija.nmaakademija.adapter;
 
 import android.content.Context;
+import android.support.v7.widget.RecyclerView;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.ArrayAdapter;
 import android.widget.TextView;
 
 import com.nmakademija.nmaakademija.R;
 import com.nmakademija.nmaakademija.entity.ScheduleDayBanner;
-import com.nmakademija.nmaakademija.entity.ScheduleItem;
 import com.nmakademija.nmaakademija.entity.ScheduleEvent;
+import com.nmakademija.nmaakademija.entity.ScheduleItem;
 
-import java.util.ArrayList;
-
-/**
- * Created by dekedro on 16.8.22.
- */
-public class ScheduleAdapter extends ArrayAdapter<ScheduleItem> {
-
-    public ScheduleAdapter(Context context, ArrayList<ScheduleItem> events){
-        super(context, R.layout.schedule_event, events);
-    }
+import java.util.List;
 
-    @Override
-    public boolean isEnabled (int position){
-        return false;
-    }
+public class ScheduleAdapter extends RecyclerView.Adapter<ScheduleAdapter.MyViewHolder> {
 
-    @Override
-    public View getView (int position, View convertView, ViewGroup parent){
-        ScheduleItem scheduleItem = getItem (position);
+    public class MyViewHolder extends RecyclerView.ViewHolder {
+        public TextView startTime, endTime, name, author;
 
-        if (scheduleItem instanceof ScheduleEvent) {
-            return getViewScheduleEvent((ScheduleEvent) scheduleItem, convertView, parent);
-        }
-        if (scheduleItem instanceof ScheduleDayBanner) {
-            return getViewScheduleDayBanner((ScheduleDayBanner) scheduleItem, convertView, parent);
+        public MyViewHolder(View view) {
+            super(view);
+            startTime = (TextView) view.findViewById(R.id.schedule_start_time);
+            endTime = (TextView) view.findViewById(R.id.schedule_end_time);
+            name = (TextView) view.findViewById(R.id.schedule_name);
+            author = (TextView) view.findViewById(R.id.schedule_author);
         }
-        return convertView;
     }
 
-    private View getViewScheduleEvent (ScheduleEvent scheduleEvent, View convertView, ViewGroup parent){
-        convertView = LayoutInflater.from(getContext()).inflate(R.layout.schedule_event, parent, false);
+    private Context _context;
```

Comment:
Don't start variable names with underscore. Camel case is good enough in these cases. More about android guidelines https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md 
P.S I wouldn't recommend to add `m` and `s` prefixes to variable names.